alignmentedas/src/main/java/eu/excitementproject/eop/alignmentedas/p1eda/classifiers/EDABinaryClassifierFromWeka.java

```diff
@@ -129,7 +127,7 @@ public DecisionLabelWithConfidence classifyInstance(
 		// Make a new instance, with the given feature vector. 
 		Instance anInstance = null; 
 		try {
-			anInstance = new Instance(featureVector.size()); 
+			anInstance = new BinarySparseInstance(featureVector.size());
```

Comment:
Not sure if this is the right concrete class to use. Options are DenseInstance, SparseInstance, BinarySparseInstance.